### ISSUE TITLE
Clustering::get_core_members: TypeError when use binary_values

### ISSUE THREAD

**Issue by rimms** [jubatus/jubatus-python-client#59]
* Python: 2.7.9
* Jubatus: 0.8.1
* jubatus-python-client: 0.8.1
* My test script: https://github.com/rimms/misc/tree/master/clustering-api

```
Traceback (most recent call last):
  File "./test.py", line 33, in <module>
    print ('get_core_members => {}'.format(client.get_core_members()))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/clustering/client.py", line 23, in get_core_members
    TUserDef(WeightedDatum))), [])
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/client.py", line 41, in call
    return ret_type.from_msgpack(ret)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 103, in from_msgpack
    return list(map(self.type.from_msgpack, m))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 103, in from_msgpack
    return list(map(self.type.from_msgpack, m))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 157, in from_msgpack
    return self.type.from_msgpack(m)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/clustering/types.py", line 23, in from_msgpack
    val = cls.TYPE.from_msgpack(arg)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 141, in from_msgpack
    tpl.append(type.from_msgpack(x))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 70, in from_msgpack
    return Datum.from_msgpack(m)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/datum.py", line 63, in from_msgpack
    val = Datum.TYPE.from_msgpack(arg)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 141, in from_msgpack
    tpl.append(type.from_msgpack(x))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 103, in from_msgpack
    return list(map(self.type.from_msgpack, m))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 141, in from_msgpack
    tpl.append(type.from_msgpack(x))
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 18, in from_msgpack
    check_types(m, self.types)
  File "/home/rimms/.pyenv/versions/2.7.9/lib/python2.7/site-packages/jubatus/common/types.py", line 11, in check_types
    raise TypeError('type %s is expected, but %s is given' % (t, type(value)))
TypeError: type <type 'str'> is expected, but <type 'unicode'> is given
```

This error does not occur with jubatus-python-client 0.8.0.

**Comment by kmaehashi** [comment]
I'll look into this.

@rimms Please raise another issue for problem that binary_values are converted into feature vector even if binary_rules are not specified.

**Comment by kmaehashi** [comment]
Fixed via #60
